fe/src/main/java/org/apache/doris/common/InternalErrorCode.java

```diff
@@ -0,0 +1,72 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.common;
+
+public class InternalErrorCode {
```

Comment:
It is strange to define a "ErrorCode" class with a `msg` in it.
How about just leave the "code" in `InternalErrorCode`, without the `msg`.

And you can put the `InternalErrorCode` into an `Exception` class.

And in routine load, you can also define a new class named `ErrorReason` with an `InternalErrorCode` and a `msg` in it.

If so, it would be more flexible to use the code everywhere we like.

Reply:
Why not just using Enum?